### PR TITLE
Add build instructions to readme file

### PR DESCRIPTION
**Pull request by Phylu**
Add some documentation about running and building mailsync that I missed when I was trying to run a linux build as well as working with command line parameters. Also removed a deprecated readme file from nylas mail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,30 @@ learn from other people doing development.
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
+### Running Mailspring from Source
+
+To run Mailspring from its source code, run the following command from the root
+directory of the Mailspring repository:
+
+```
+npm start
```
Comment (CodeMouse92) [inline README.md:53]: One needs to run `npm install` first, at least the first time (and after each git pull).

**Review by CodeMouse92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Phylu) [reply README.md:53]: Added this.

**Review by CodeMouse92 — COMMENTED** [review]
Awesome! (Now we just need Ben's approval.)

**Review by CodeMouse92 — APPROVED** [review]

**Comment by bengotow** [comment]
This looks great, thanks @Phylu!

**Pull request merged**